ui/components/MesheryPatterns.js

```diff
@@ -880,7 +880,7 @@ function MesheryPatterns({
     try {
       downloadFile({ id, name, type : "pattern" })
       updateProgress({ showProgress : false });
-      notify({ message : `"${name}" Design Downloaded` , event_type : EVENT_TYPES.SUCCESS })
+      notify({ message : `"${name}" Design Downloaded` , event_type : EVENT_TYPES.INFO })
```

Comment:
@Rajdip019 let's be consistent with capitalization or lack thereof across messages.

Reply:
Sorry missed it. Thanks for mentioning. I will try to be more attentive from now on.